src/run.test.ts

```diff
@@ -21,15 +21,15 @@ describe("Set context", () => {
       if (inputName == "resource-group") return resourceGroup;
     });
     await expect(run()).rejects.toThrow();
-  });
+  }, 17500);
 
   it("throws without az tools", async () => {
     jest.spyOn(core, "getInput").mockImplementation((inputName, options) => {
       if (inputName == "resource-group") return resourceGroup;
       if (inputName == "cluster-name") return clusterName;
     });
     await expect(run()).rejects.toThrow();
-  });
+  }, 17500);
```

Comment:
can you extract 17500 to a constant and maybe leave a comment with some context about the GitHub runner acting up?

Reply:
done